apps/crn-frontend/src/network/users/UserProfile.tsx

```diff
@@ -78,6 +78,18 @@ const UserProfile: FC<UserProfileProps> = ({ currentTime }) => {
     currentUser,
   );
 
+  const pastEventsResult = useEvents(
```

Comment:
same as the team profile, can we have `useEvents` return all the events? is it bad for performance?